containers/pom.xml

```diff
@@ -112,8 +108,8 @@
       </activation>
       <modules>
         <module>cassandra-seed-provider</module>
-        <module>hawkular-metrics-openshift-jaxrs-1.1-integration</module>
-        <module>metrics-api-jaxrs-1.1-openshift</module>
+        <module>hawkular-metrics-openshift-integration</module>
+        <module>metrics-api-jaxrs-openshift</module>
       </modules>
```

Comment:
What's the difference between the `openshift` profile and the `eap-containers` profile? Shouldn't we keep just one?